easybuild/easyconfigs/c/CP2K/CP2K-2.5.1-intel-2014b-psmp.eb

```diff
@@ -0,0 +1,42 @@
+name = 'CP2K'
+version = '2.5.1'
+versionsuffix = '-psmp'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [SOURCEFORGE_SOURCE]
+
+patches = [
+    'CP2K-20131211-ifort-compiler-bug-fix.patch',
+    'CP2K-2.4.0-fix_compile_date_lastsvn.patch',
+]
+
+dependencies = [
+    ('libint2', '2.0.3'),
+    ('libxc', '2.2.0'),
+    ('FFTW', '3.3.4'),
```

Comment:
I think this is me trying to fix the problem I was running into, it had something to do with linking to FFTW...